spring-vertx/src/main/java/com/baeldung/vertxspring/config/PortConfiguration.java

```diff
@@ -0,0 +1,32 @@
+package com.baeldung.vertxspring.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+
+@Configuration
+public class PortConfiguration {
+
+    private static final int DEFAULT_PORT = 8069;
+
```

Comment:
Hey @kwoyke - One very minor thing, using the eclipse formatter, we lose one of the empty lines. Sorry but let's fix that.